src/modules/OneClickInstallStandard.rb

```diff
@@ -42,85 +46,55 @@ def main
     #              ],
     #      ]
     def GetRepositoriesFromXML(filename)
-      ret = []
 
       if !FileUtils.Exists(filename)
```

Comment:
NP: I can see a small inconsistency here. The `FileUtils.Exists` honors the SCR target chroot while the later `File.read` not. But because this is used only in an installed system we do not need to care about possible chroot.

So I'd simply replace `FileUtils.Exists` with standard Ruby `File.exist?`.

Reply:
yes, possible. So lets do it.